src/main/java/Ui.java

```diff
@@ -0,0 +1,79 @@
+public class Ui {
+    public Ui() {
+    }
+
+    /**
+     * Displays welcome message.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Jia Le :)");
+        System.out.println("How may I be of assistance to you?");
+    }
+
+    /**
+     * Shows system output when user inputs "list" command.
+     * @param tasks is the TaskList containing pre-existing tasks.
+     */
+    public void list(TaskList tasks) {
+        int count = 1;
+        for (Task task : tasks.getList()) {
+            System.out.println(count + "." + task.toString());
+            count++;
+        }
+    }
+
+    /**
+     * Shows system output when user inputs "done" command.
+     * @param tasks is the TaskList containing pre-existing tasks.
+     * @param index represents the index of task in list that is completed.
+     */
+    public void done(TaskList tasks, int index) {
```

Comment:
The name of this method isn't unclear about what it's doing, but it doesn't quite fit the format of being a verb. Maybe `markAsDone` would be better? `list` could also be renamed to `showList`, and `add` could be more specific with what's being added to where.